[2025-09-11T05:53:29.015+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: aviationstack_kafka_dag.run_kafka_consumer manual__2025-09-11T05:53:27.831266+00:00 [queued]>
[2025-09-11T05:53:29.017+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: aviationstack_kafka_dag.run_kafka_consumer manual__2025-09-11T05:53:27.831266+00:00 [queued]>
[2025-09-11T05:53:29.017+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-09-11T05:53:29.021+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): run_kafka_consumer> on 2025-09-11 05:53:27.831266+00:00
[2025-09-11T05:53:29.022+0000] {standard_task_runner.py:60} INFO - Started process 4055 to run task
[2025-09-11T05:53:29.024+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'aviationstack_kafka_dag', 'run_kafka_consumer', 'manual__2025-09-11T05:53:27.831266+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/aviationstack_dag.py', '--cfg-path', '/tmp/tmp10yv1z68']
[2025-09-11T05:53:29.025+0000] {standard_task_runner.py:88} INFO - Job 24: Subtask run_kafka_consumer
[2025-09-11T05:53:29.031+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.11/site-packages/airflow/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-09-11T05:53:29.042+0000] {task_command.py:423} INFO - Running <TaskInstance: aviationstack_kafka_dag.run_kafka_consumer manual__2025-09-11T05:53:27.831266+00:00 [running]> on host 81f5a06342ec
[2025-09-11T05:53:29.069+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='aviationstack_kafka_dag' AIRFLOW_CTX_TASK_ID='run_kafka_consumer' AIRFLOW_CTX_EXECUTION_DATE='2025-09-11T05:53:27.831266+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-11T05:53:27.831266+00:00'
[2025-09-11T05:53:29.069+0000] {kafka_consumer.py:90} INFO - [DEBUG] Using existing Kafka payload: data/kafka_logs/as_message_20250911_020013.json
[2025-09-11T05:53:29.069+0000] {python.py:201} INFO - Done. Returned value was: data/kafka_logs/as_message_20250911_020013.json
[2025-09-11T05:53:29.076+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=aviationstack_kafka_dag, task_id=run_kafka_consumer, execution_date=20250911T055327, start_date=20250911T055329, end_date=20250911T055329
[2025-09-11T05:53:29.122+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-11T05:53:29.132+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
